src/games/strategy/engine/framework/ui/NewGameChooser.java

```diff
@@ -109,9 +110,7 @@ public static NewGameChooserEntry chooseGame(final Frame parent, final String de
     NewGameChooser chooser = new NewGameChooser(parent);
     chooser.setSize(800, 600);
     chooser.setLocationRelativeTo(parent);
-    if (defaultGameName != null) {
-      chooser.selectGame(defaultGameName);
-    }
+    chooser.selectGame(defaultGameName);
```

Comment:
I'm curious whether it is `chooser.selectGame` handles the NPE situation, or whether `defaultGameName` is never null?